Package/Sublime Text Commands/Sublime Text Commands.sublime-syntax

```diff
@@ -68,6 +75,33 @@ contexts:
         3: punctuation.definition.string.end.json
       set: [expect-any-value, expect-colon]
 
+  expect-platform-string-value:
+    - match: (?=")
+      set: [mapping-value-meta, platform-name-string-pop]
+    - include: expect-string-rest
+
+  platform-name-string-pop:
+    - match: (\")(!)?(Windows|Linux|OSX)(\")
+      scope: meta.platform-name.sublime-commands string.quoted.double.json
+      captures:
+        1: punctuation.definition.string.begin.json
+        2: keyword.operator.logical.sublime-commands
+        3: support.constant.command-name.sublime-commands
+        4: punctuation.definition.string.end.json
+      pop: true
+    - match: \" # probably illegal command name
+      scope: punctuation.definition.string.begin.json
+      set:
+        - meta_scope: meta.platform-name.sublime-commands string.quoted.double.json
+        - meta_content_scope: invalid.illegal.invalid-command.sublime-commands
```

Comment:
Incorrect scope.

Reply:
Can you tell which is the incorrect scope. I can then add a commit that rectifies it for both syntaxes.

Reply:
`invalid-command` should be `invalid-platform`